src/EnergyPlus/Coils/CoilCoolingDXCurveFitSpeed.cc

```diff
@@ -309,14 +310,15 @@ CoilCoolingDXCurveFitSpeed::CoilCoolingDXCurveFitSpeed(EnergyPlus::EnergyPlusDat
         input_specs.gross_rated_cooling_COP = state.dataIPShortCut->rNumericArgs(5);
         input_specs.active_fraction_of_coil_face_area = state.dataIPShortCut->rNumericArgs(6);
         input_specs.rated_evaporator_fan_power_per_volume_flow_rate = state.dataIPShortCut->rNumericArgs(7);
-        input_specs.rated_evaporative_condenser_pump_power_fraction = state.dataIPShortCut->rNumericArgs(8);
-        input_specs.evaporative_condenser_effectiveness = state.dataIPShortCut->rNumericArgs(9);
+        input_specs.rated_evaporator_fan_power_per_volume_flow_rate_2023 = state.dataIPShortCut->rNumericArgs(8);
+        input_specs.rated_evaporative_condenser_pump_power_fraction = state.dataIPShortCut->rNumericArgs(9);
+        input_specs.evaporative_condenser_effectiveness = state.dataIPShortCut->rNumericArgs(10);
```

Comment:
Mapping the value from the input processor, through the input structure, and onto the coil.  OK.